package.json

```diff
@@ -13,21 +13,21 @@
     "prepare": "patch-package"
   },
   "dependencies": {
-    "@ecl-twig/ec-components": "2.35.2",
-    "@ecl/builder": "2.35.0",
-    "@ecl/ec-component-fact-figures": "2.35.0",
-    "@ecl/ec-component-form-help-block": "2.35.0",
-    "@ecl/ec-component-media-container": "2.35.0",
-    "@ecl/ec-preset-editor": "2.35.0",
-    "@ecl/ec-preset-website": "2.35.0",
-    "@ecl/ec-theme-default": "2.35.0",
-    "@ecl/ec-utility-background": "2.35.0",
-    "@ecl/ec-utility-border": "2.35.0",
-    "@ecl/eu-component-form-help-block": "2.35.0",
-    "@ecl/eu-preset-website": "2.35.0",
-    "@ecl/eu-theme-default": "2.35.0",
-    "@ecl/eu-utility-background": "2.35.0",
-    "@ecl/eu-utility-border": "2.35.0",
+    "@ecl-twig/ec-components": "2.37.0",
+    "@ecl/builder": "2.37.0",
+    "@ecl/ec-component-fact-figures": "2.37.0",
+    "@ecl/ec-component-form-help-block": "2.37.0",
+    "@ecl/ec-component-media-container": "2.37.0",
+    "@ecl/ec-preset-editor": "2.37.0",
+    "@ecl/ec-preset-legacy-website": "2.37.0",
+    "@ecl/ec-theme-default": "2.37.0",
+    "@ecl/ec-utility-background": "2.37.0",
+    "@ecl/ec-utility-border": "2.37.0",
+    "@ecl/eu-component-form-help-block": "2.37.0",
+    "@ecl/eu-preset-legacy-website": "2.37.0",
```

Comment:
Legacy components are back, we should keep the non legacy.

Reply:
Fixed.